econf/econf.py

```diff
@@ -268,41 +268,41 @@ def read_dirs_history(
             err, f"read_dirs_history failed with error: {err_string(err)}"
         )
     arr = cast(key_files, POINTER(c_void_p * c_size.value))
-    result = [Keyfile(c_void_p(i)) for i in arr.contents]
+    result = [EconfFile(c_void_p(i)) for i in arr.contents]
     return result
 
 
-def new_key_file(delim: str | bytes, comment: str | bytes) -> Keyfile:
+def new_key_file(delim: str | bytes, comment: str | bytes) -> EconfFile:
     """
     Create a new empty keyfile
 
     :param delim: delimiter of a key/value e.g. '='
     :param comment: string that defines the start of a comment e.g. '#'
-    :return: created Keyfile object
+    :return: created EconfFile object
     """
-    result = Keyfile(c_void_p)
+    result = EconfFile(c_void_p)
     delim = _encode_str(delim)
     comment = _encode_str(comment)
-    err = LIBECONF.econf_newKeyFile(byref(result.ptr), delim, comment)
+    err = LIBECONF.econf_newKeyFile(byref(result.__ptr), delim, comment)
     if err:
         raise _exceptions(err, f"new_key_file failed with error: {err_string(err)}")
     return result
 
 
-def new_ini_file() -> Keyfile:
+def new_ini_file() -> EconfFile:
     """
     Create a new empty keyfile with delimiter '=' and comment '#'
 
-    :return: created Keyfile object
+    :return: created EconfFile object
     """
-    result = Keyfile(c_void_p)
-    err = LIBECONF.econf_newIniFile(byref(result.ptr))
+    result = EconfFile(c_void_p())
+    err = LIBECONF.econf_newIniFile(byref(result.__ptr))
     if err:
         raise _exceptions(err, f"new_ini_file failed with error: {err_string(err)}")
     return result
 
 
-def write_file(kf: Keyfile, save_to_dir: str, file_name: str) -> Econf_err:
+def write_file(kf: EconfFile, save_to_dir: str, file_name: str) -> Econf_err:
```

Comment:
Do not kill me, but maybe now kf should be ef,  key_files should be econf_files, etc : D

Reply:
No problem, I will update it